docker/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2
+FROM python:3
```

Comment:
Because 3.6 is not available on Debian stable, and we want the same version every where :-)